EIPS/eip-x.md

```diff
@@ -0,0 +1,292 @@
+---
+eip: PENDING
```

Comment:
```suggestion
eip: 7748
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.